.golangci.yaml

```diff
@@ -0,0 +1,29 @@
+# golangci-lint configuration file
+# see: https://golangci-lint.run/usage/configuration/
+
+# Options for analysis running
+run:
+  # Which dirs to skip: they won't be analyzed;
+  skip-dirs:
+    - bin
+
+# Settings of specific linters
+linters-settings:
+  goimports:
+    local-prefixes: sigs.k8s.io/kueue
+
+# Settings for enabling and disabling linters
+linters:
+  disable-all: true
+  enable:
+    - deadcode
+    - errcheck
+    - goimports
+    - gosimple
+    - govet
+    - ineffassign
+    - staticcheck
+    - structcheck
+    - typecheck
+    - unused
+    - varcheck
```

Comment:
These linters are automatically enabled by default except for goimports, so the actual change is simply enabling it
- https://golangci-lint.run/usage/linters/#enabled-by-default